### ISSUE TITLE
[Keyboard Navigation-MCIO DCX-'Design & Engineering SoS' page]: 'Select all' checkbox of the tables present under 'Design& Engineering SoS' are not accessible through keyboard.

### ISSUE THREAD

**Issue by msftedad** [angular-ui/ui-grid#7243]
**Description**: Users who rely on the keyboard will be affected if the checkbox elements are not accessible with the tab key or arrow keys. Keyboard users cannot land on it and hence cannot use the functionality.

**Environment (OS, Application, Versions)**:
**Test Environment**:
OS Build: Version 21H2(22000.856)
Browser: Microsoft Edge Version 105.0.1343.25 (Official build) (64-bit) 
Screen reader: Non-Visual Desktop Access (NVDA) (2022.2)

**Repro Steps**: 
1. Open "[[Angular Grid Row Selection | Ignite UI for Angular | infragistics](https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/row-selection)](https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/row-selection)"
2. Angular Row selection row pen page will open.
3. Reach to the main landmark content where ' Examples' tab is selected.
4. With 6-tab press, reach to the column header of the available table.
5. Verify the issue that the 'select all' check box control is keyboard accessible or not.

**Actual Results**: 
'Select all' checkboxes of the tables present under 'Design& Engineering SoS' are not accessible through the keyboard.

**Expected Results**: 
'Select all' checkboxes of the tables present under 'Design& Engineering SoS' should be accessible through the keyboard.

**Additional Information**:
Same Issue Reproes with 'select all' checkbox of the tables present throughout the application.

**Attachment**:
[MAS2.1.1-MCIO DCX-Design & Engineering-select all (1).zip](https://github.com/angular-ui/ui-grid/files/9486816/MAS2.1.1-MCIO.DCX-Design.Engineering-select.all.1.zip)

**Comment by mportuga** [comment]
This defect is not for UI-Grid, but for Ignite UI, which is managed by a completely different team.
